packages/corvid-local-site/src/sitePaths.js

```diff
@@ -54,6 +56,21 @@ const matchLocalPageFile = extension => filePath => {
     : null;
 };
 
+const matchLocalPageTsConfigFile = filePath => {
+  const matches = filePath.match(
+    new RegExp(
+      `^(?<root>${ROOT_PATHS.PAGES}|${
```

Comment:
@nemni8 this regex repeats in multiple places. let's create a function that only get the base file name and returns this regex